trade-finance-manager-ui/server/controllers/case/parties/index.js

```diff
@@ -39,87 +47,304 @@ const getCaseParties = async (req, res) => {
   });
 };
 
-const getPartyDetails = (partyType) => (
-  async (req, res) => {
-    const { user } = req.session;
+/**
+ * Renders party specific URN edit page
+ * @param {Object} req Express request
+ * @param {Object} res Express response
+ * @returns {Object} Express response as rendered party page.
+ */
+const getPartyDetails = async (req, res) => {
+  if (!req) {
+    console.error('Invalid request received');
+    return res.redirect('/not-found');
+  }
```

Comment:
If this controller is only being called in express router I don't believe we need this check, would be like testing that express is working